phoenix-pherf/pom.xml

```diff
@@ -152,6 +152,12 @@
     </dependency>
 
     <!-- Test Dependencies -->
+    <dependency>
+      <groupId>jakarta.xml.bind</groupId>
+      <artifactId>jakarta.xml.bind-api</artifactId>
+      <version>2.3.2</version>
```

Comment:
Can you please define this dependency version on parent pom?

Reply:
We already depend on **javax.xml.bind:bind-api**.
We should probably replace that dependency, instead of adding a new one.

Reply:
Thanks for the review guys. Sure, I'll update the PR.

Reply:
Replacing the dependeny doesn't actually work, unfortunately.

Reply:
This is dependency-plugin getting even more confused than usual by multiple sources of jaxb-api.
As pulling in the same set of classes from two packages just to appease dependecy plugin is not the way to go, we should just ignore this.

Reply:
By ignore this, I mean tell dependeny-plugin to ignore these packages.